File: TP1/TP1 python/Tomas/Ejericicio 2/Producto.py

class Producto:
    def __init__(self, nombre, dosis,gramaje, precio):
        self.nombre = nombre
        self.dosis = dosis
        self.gramaje = gramaje
        self.precio = precio
        
    
    def getPrecio(self):
        return self.precio
    
    
    def imprimir(self):
        print("{}    {}{}              {}".format(self.nombre, self.dosis, self.gramaje, self.precio))

File: TP5/Ej 9 (Decorator)/Csar/punto9/src/Producto.java
public class Producto {
    private String nombre;
    private Double precio;
    public Producto(String nombre, Double precio){
        this.nombre = nombre;
        this.precio = precio;
    }
    public String getNombre() {
        return nombre;
    }
    public String getPrecio(){
        return String.format("%.2f",precio);
        
    }

}
